src/ucp/proto/proto_am.inl

```diff
@@ -20,20 +20,22 @@
 
 #define UCP_STATUS_PENDING_SWITCH (UCS_ERR_LAST - 1)
 
-#define UCP_AM_BCOPY_HANDLE_STATUS(_multi, _status) \
+#define UCP_AM_BCOPY_HANDLE_STATUS(_multi, _status, _exit_action) \
```

Comment:
can you pls document this macro: parameters, in which cases it calls _exit_action, etc

Reply:
done